### DISCUSSION TITLE
[PSD12] Esercizio Slide 1

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#175] [Esercizi (A-L)]
Supponendo $t_{cd} = t_{pd}$, calcolare la frequenza del segnale CLK

![image](https://user-images.githubusercontent.com/69389461/210062308-91be1966-1854-4697-bae9-86fdbec9a35d.png)

![image](https://user-images.githubusercontent.com/69389461/210062282-d2e3f17c-cb4e-48d1-8c94-e35a211a0f31.png)

**Comment by Elia-Belli** [comment]
Se il periodo del CLK è di $6ns$ possiamo eseguire $10^9/6 = 0.17 \times 10^9$ operazioni al secondo circa.
Quindi abbiamo una frequenza di circa $0.17 \space GHz$

**Comment by fraxb04** [comment]
corretto
